External/FEXCore/Source/Interface/Core/Dispatcher/X86Dispatcher.cpp

```diff
@@ -418,13 +418,11 @@ X86Dispatcher::X86Dispatcher(FEXCore::Context::ContextImpl *ctx, const Dispatche
 
 }
 
-// Used by GenerateGDBPauseCheck, GenerateInterpreterTrampoline
-static thread_local Xbyak::CodeGenerator emit(1, &emit); // actual emit target set with setNewBuffer
```

Comment:
Can we not have thread_locals anymore, or is this merely not needed here?

Reply:
Merely not needed.